docs/reference/release-notes/7.11.asciidoc

```diff
@@ -1,3 +1,85 @@
+== {es} version 7.11.1
+
+coming::[7.11.1]
+
+
+Also see <<breaking-changes-7.11,Breaking changes in 7.11>>.
+
+[[enhancement-7.11.1]]
+[float]
+=== Enhancements
+
+Features/Indices APIs::
+* Prevent concurrent rollover race conditions {es-pull}67953[#67953] (issues: {es-issue}64921[#64921], {es-issue}67836[#67836])
+
+Features/Stats::
+* Add index creation version stats to cluster stats {es-pull}68141[#68141]
+
+Snapshot/Restore::
+* Improve shard evictions in searchable snapshot cache service {es-pull}67160[#67160] (issues: {es-issue}66730[#66730], {es-issue}66958[#66958])
+* Make blob cache have async durability {es-pull}66687[#66687]
+
+
+
+[[bug-7.11.1]]
+[float]
+=== Bug fixes
+
+Authentication::
+* Fix permissions for UnboundID LDAP SDK {es-pull}68872[#68872] (issues: {es-issue}64743[#64743], {es-issue}68838[#68838])
+
+Cluster Coordination::
+* Determine data role based on convention {es-pull}66933[#66933] (issue: {es-issue}66000[#66000])
+
+Features/ILM+SLM::
+* Check if delete phase has actions {es-pull}66664[#66664] (issue: {es-issue}65847[#65847])
+* Fix handling of configurtions with a combination of both maximum and expire-after retention {es-pull}68352[#68352]
+* Fix searchable snapshot action to wait for the restored index {es-pull}68309[#68309] (issue: {es-issue}67879[#67879])
+* Ignore `clusterChanged` events if state not recovered {es-pull}67507[#67507] (issue: {es-issue}67506[#67506])
+* `ShrinkStep` ignores the `isAcknowledged` flag on the response {es-pull}67591[#67591] (issue: {es-issue}53111[#53111])
+* Skip rollover if the data stream is rolled over already {es-pull}67778[#67778] (issue: {es-issue}67777[#67777])
+
+Features/Indices APIs::
+* Fix exception when validating composable template with no template {es-pull}67310[#67310] (issue: {es-issue}66949[#66949])
+
+Features/Ingest::
+* Fix parsing empty value with brackets failed in dissect ingest processor {es-pull}65524[#65524] (issue: {es-issue}65080[#65080])
+
+Features/Stats::
+* Allow reading of `/proc/meminfo` for JDK bug workaround {es-pull}68742[#68742]
+* Workaround for JDK bug with total memory on Debian 8 {es-pull}68542[#68542] (issues: {es-issue}66629[#66629], {es-issue}66885[#66885])
+
+Geo::
+* Add support for `distance_feature` query for runtime `geo_point` field {es-pull}68094[#68094] (issue: {es-issue}67910[#67910])
+
+Infra/Core::
+* Fix an error that occurs while starting the service in Windows OS under a path with a space in it {es-pull}61895[#61895] (issue: {es-issue}61891[#61891])
+
+Infra/Scripting::
+* Remove leniency for casting from a `def` type to a `void` type in Painless {es-pull}66957[#66957] (issue: {es-issue}66175[#66175])
+
+Machine Learning::
+* Ensure mappings are up to date before reverting state {es-pull}68746[#68746]
+
+Network::
+* Fix `AbstractClient#execute` Listener Leak {es-pull}65415[#65415] (issue: {es-issue}65405[#65405])
+
+Packaging::
+* Fix installation using Windows MSI (issue: {es-issue}68914[#68914])
```

Comment:
@jrodewig I added this for https://github.com/elastic/elasticsearch/issues/68914 / https://github.com/elastic/windows-installers/issues/402

Reply:
Thanks @pugnascotia! I'll reach out offline.

Reply:
@jrodewig a fix is merged - I've linked to the PR in the relevant repo.